daemon/containerd/image_history.go

```diff
@@ -161,3 +163,24 @@ func getImageTags(ctx context.Context, imgs []images.Image) []string {
 
 	return tags
 }
+
+// getParentsByBuilderLabel finds images that were a base for the given image
```

Comment:
This godoc uses the plural (e.g. "finds image**s**") and we do return the result of `ImageService().List()`, but in general this should result in "one" image right? (the direct parent)

Reply:
It will be one image in terms of content, but the actual tag might be different (`busybox:latest`, `busybox:3.16`, etc).

It's not new in this PR though, it was just moved to a different file